lms/templates/courseware/courseware.html

```diff
@@ -45,7 +46,7 @@
     % endfor
 % endif
 
-% if include_special_exams:
+% if include_special_exams is not UNDEFINED:
```

Comment:
What if `include_special_exams` is not undefined but `False`? We could do `include_special_exams is not UNDEFINED and include_special_exams`